redbot/core/commands/commands.py

```diff
@@ -610,6 +610,27 @@ def enable_in(self, guild: discord.Guild) -> bool:
         else:
             return True
 
+    def is_disabled_in_guild(self, guild: discord.Guild) -> bool:
+        """Checks if this command is disabled in the given guild.
+
+        Parameters
+        ----------
+        guild : discord.Guild
+            The guild to enable the command in.
+
+        Returns
+        -------
+        bool
+            ``True`` if the command is disabled.
+            ``False`` if the command isn't disabled.
+
+        """
```

Comment:
```suggestion
        -------
        bool
            ``True`` if the command is disabled.

        """
```
False is a bit redundant